charts/beyla/values.yaml

```diff
@@ -186,6 +186,21 @@ config:
     attributes:
       kubernetes:
         enable: true
+      select:
+        beyla_network_flow_bytes:
+          include:
+            - 'k8s.src.owner.type'
+            - 'k8s.dst.owner.type'
+            - 'direction'
+    filter:
+      network:
+        k8s_dst_owner_name:
+          not_match: '{kube*,*jaeger-agent*,*prometheus*,*promtail*,*grafana-agent*}'
+        k8s_src_owner_name:
+          not_match: '{kube*,*jaeger-agent*,*prometheus*,*promtail*,*grafana-agent*}'
```

Comment:
These are k8s infra and monitoring components that has many connections, but not quite useful for o11y use cases, so we are suggesting to exclude them by default.